website/src/libs/api-v3/password.js

```diff
@@ -1,5 +1,7 @@
+// Utilities for working with passwords
```

Comment:
I'm conflicted about comments like this. Isn't it obvious that this is a lib about passwords?